packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -1167,116 +1129,6 @@ Generated by [AVA](https://ava.li).
-    ␊
-    > Current directory␊
-      packages/build/tests/core/fixtures/local_node_path␊
-    ␊
-    > Config file␊
-      packages/build/tests/core/fixtures/local_node_path/netlify.toml␊
-    ␊
-    > Resolved config␊
-      build:␊
-        publish: packages/build/tests/core/fixtures/local_node_path␊
-        publishOrigin: default␊
-      plugins:␊
-        - inputs: {}␊
-          origin: config␊
-          package: ./plugin.js␊
-    ␊
-    > Context␊
-      production␊
-    ␊
-    > Loading plugins␊
-       - ./plugin.js@1.0.0 from netlify.toml␊
-    ␊
-    ────────────────────────────────────────────────────────────────␊
-      1. onPreBuild command from ./plugin.js                        ␊
-    ────────────────────────────────────────────────────────────────␊
-    ␊
-    expect execPath to equal TEST_NODE_PATH. Got 'true'␊
-    ␊
-    (./plugin.js onPreBuild completed in 1ms)␊
-    ␊
-    ────────────────────────────────────────────────────────────────␊
-      Netlify Build Complete                                        ␊
-    ────────────────────────────────────────────────────────────────␊
-    ␊
-    (Netlify Build completed in 1ms)`
-
-## --node-path is used by plugins added to package.json if the feature flag plugins_system_node_version is set to false
-
-> Snapshot 1
-
-    `␊
-    ────────────────────────────────────────────────────────────────␊
-      Netlify Build                                                 ␊
-    ────────────────────────────────────────────────────────────────␊
-    ␊
-    > Version␊
-      @netlify/build 1.0.0␊
-    ␊
-    > Flags␊
-      debug: true␊
-      nodePath: /external/path␊
-      repositoryRoot: packages/build/tests/core/fixtures/package␊
-      testOpts:␊
-        pluginsListUrl: test␊
-        silentLingeringProcesses: true␊
-    ␊
-    > Current directory␊
-      packages/build/tests/core/fixtures/package␊
-    ␊
-    > Config file␊
-      packages/build/tests/core/fixtures/package/netlify.toml␊
-    ␊
-    > Resolved config␊
-      build:␊
-        publish: packages/build/tests/core/fixtures/package␊
-        publishOrigin: default␊
-      plugins:␊
-        - inputs: {}␊
-          origin: config␊
-          package: netlify-plugin-test␊
-    ␊
-    > Context␊
-      production␊
-    ␊
-    > Loading plugins␊
-       - netlify-plugin-test@1.0.0 from netlify.toml and package.json␊
-    ␊
-    ────────────────────────────────────────────────────────────────␊
-      1. onPreBuild command from netlify-plugin-test                ␊
-    ────────────────────────────────────────────────────────────────␊
-    ␊
-    expect execPath to equal TEST_NODE_PATH. Got 'true'␊
-    ␊
-    (netlify-plugin-test onPreBuild completed in 1ms)␊
-    ␊
-    ────────────────────────────────────────────────────────────────␊
-      Netlify Build Complete                                        ␊
-    ────────────────────────────────────────────────────────────────␊
-    ␊
-    (Netlify Build completed in 1ms)`
-
```

Comment:
This cases are no longer applicable, they were the negation of the FF, so we validated that we were still using the `--node-path` the user choose (v8 in this particular case).